README.md

```diff
@@ -59,5 +59,7 @@ In order to run the full suite of Acceptance tests, run `make testacc`.
 *Note:* Acceptance tests create real resources, and often cost money to run.
```

Comment:
Could you just add a statement that clearly explain that Docker and Docker Compose are needed tools in order to run tests?

Reply:
Sure, I'll add the clarification. Regarding your other comment: no worries I can provide both the functionalities :)